docs/deploying_apps/jenkins.md

```diff
@@ -56,7 +56,7 @@ Further information can be found on the [Cloud Foundry plugin's wiki page](https
 
 Before you do this, make sure you first [set up the credentials plugin](/deploying_apps/jenkins/#setting-up-the-credentials-plugin).
 
-Note that currently, this approach briefly exposes the password via the process command line. If this risk is not acceptable, please use the Cloud Foundry plugin described above. The Cloud Foundry project is aware of the problem and we expect they will provide a more secure login mechanism soon.
+Note that using the custom scripts approach exposes the password via the process command line, so it can be read by other processes running on the same machine. If this risk is not acceptable, please use the Cloud Foundry plugin described above. The Cloud Foundry project is aware of the problem and we expect they will provide a more secure login mechanism soon.
```

Comment:
I think we should mention that it can be workaround by using `echo password | cf login -u ....`